language/diem-framework/modules/0L/Migrations.move

```diff
@@ -106,4 +106,22 @@ module MigrateTowerCounter {
   }
 
 }
+
+// Module for initializing Teams on a hot upgrade of stdlib.
+// since the system is likely operating and Teams are introduced as an upgrade, the structs need to be initalized.
+
+module MigrateInitDelegation {
+  use 0x1::Teams;
+  // use 0x1::TowerState;
+  use 0x1::Migrations;
+  const UID: u64 = 2;
+  public fun do_it(vm: &signer) {
+    if (!Migrations::has_run(UID)) {
+      Teams::vm_init(vm);
+      // also initialize relevant state in TowerState
+      // TowerState::init_team_thresholds(vm);
+      Migrations::push(vm, UID, b"MigrateInitTeams");
```

Comment:
CoreAddresses::assert_vm(sender); - if we need this check here

Reply:
Good catch, let me check.

Reply:
fixed with 3922dc5afded960d305ab990ba0c1e60959eea6c